src/Symfony/Component/Filesystem/Filesystem.php

```diff
@@ -128,6 +128,26 @@ public function exists($files)
     }
 
     /**
+     * Checks if files are real files.
+     *
+     * @param string|array|\Traversable $files A filename, an array of files, or a \Traversable instance to check
+     *
+     * @return bool true if the file is a real file, false otherwise
+     */
+    public function isFile($files)
+    {
+        foreach ($this->toIterator($files) as $file) {
```

Comment:
Yes I know, but it's the same feature used for `file_exists`, so it must be corrected in this function too. 5I've just made a copy-paste, changing the function used to check)